### ISSUE TITLE
bottom panes appear to overflow into the status line after upgrading from tmux 2.8 to 2.9

### ISSUE THREAD

**Issue by halfmu** [jmaslak/Term-Tmux-Layout#2]
After upgrading from tmux 2.8 to 2.9, whenever I create a vertical split using tmuxlayout, the bottom pane will overflow into the status bar for me.  If I uncomment the following line however, it would work fine.

https://github.com/jmaslak/Term-Tmux-Layout/blob/f4c41f0fc69fd6775d493196e12cf5ecb64b4a5a/lib/Term/Tmux/Layout.pm#L195-L196

**Comment by jmaslak** [comment]
Interesting. I'll see if I can get my head around what I commented those lines out and take a look.

**Comment by jmaslak** [comment]
I figured out why I commented those lines out (a misguided attempt to mimic how tmux splits windows), but that was clearly the wrong thing to do and has been wrong since 2015.  If you want to test the code now, I'd love verification it now works for you.  This fix is in the master branch or in the release I just pushed to CPAN (the *TRIAL* release - Term-Tmux-Layout-1.192420-TRIAL.tar.gz - which should be available within a few hours.
